copter/source/docs/precision-landing-with-irlock.rst

```diff
@@ -1,23 +1,59 @@
 .. _precision-landing-with-irlock:
 
-=========================================
-Precision Landing and Loiter with IR-LOCK
-=========================================
+============================
+Precision Landing and Loiter
+============================
 
 Overview
 ========
 
-Copter supports Precision Landing using the `IR-LOCK sensor <https://irlock.com/collections/frontpage/products/ir-lock-sensor-precision-landing-kit>`__ and a :ref:`sonar or lidar <common-rangefinder-landingpage>`.
-Using this system, when the vehicle enters LAND mode (and has GPS lock) it is possible to reliably land within 30cm of an IR beacon that is moving at less than 1m/s.
+Copter supports Precision Landing using the `IR-LOCK sensor <https://irlock.com/collections/frontpage/products/ir-lock-sensor-precision-landing-kit>`__ and a :ref:`sonar or lidar <common-rangefinder-landingpage>`, or via `MAVLink LANDING_TARGET messages <https://mavlink.io/en/messages/common.html#LANDING_TARGET>`__ from a companion computer providing position differences from a target such as an `April tag <https://docs.wpilib.org/en/stable/docs/software/vision-processing/apriltag/apriltag-intro.html>`__.
 
-Copter also supports Precision Loiter which allows a vehicle to maintain its position above a target while in Loiter mode.  The Pilot can enable this using one of the transmitter's :ref:`auxiliary function switches <common-auxiliary-functions>` (in versions prior to Copter-4.0, CH7_OPT or CH_8_OPT would be used).
+Using this system, when the vehicle enters LAND mode (and has GPS lock) it is possible to reliably land within 30cm of an IR beacon that is moving at less than 1m/s (unless :ref:`PLND_OPTIONS<PLND_OPTIONS>` bit 0 is set to allow a moving landing target) .
```

Comment:
GPS lock isn't accurate.. I think we should replace this with "Has position aiding source, such as GPS or Optical Flow"). I do precision landing with just optical flow sensors without GPS all the time